src/main/java/org/liliya/hotelapp/ui/Console.java

```diff
@@ -0,0 +1,119 @@
+package org.liliya.hotelapp.ui;
+
+import org.liliya.hotelapp.exception.ReservationException;
+import org.liliya.hotelapp.model.Apartment;
+import org.liliya.hotelapp.model.Client;
+import org.liliya.hotelapp.service.ApartmentService;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Scanner;
+
+public class Console {
+    private final ApartmentService apartmentService;
+    private final Scanner scanner;
+
+    public Console(ApartmentService apartmentService) {
+        this.apartmentService = apartmentService;
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void start() {
+        while (true) {
+            try {
+                showMenu();
+                int choice = scanner.nextInt();
+                scanner.nextLine();
+                switch (choice) {
+                    case 1 -> registerApartment();
+                    case 2 -> reserveApartment();
+                    case 3 -> releaseApartment();
+                    case 4 -> getPaginatedAndSortedApartments();
+                    case 5 -> {
+                        return;
+                    }
+                    default -> System.out.println("\nInvalid option. Please try again.");
+                }
+            } catch (ReservationException e) {
+                System.out.println("Reservation failed: " + e.getMessage());
+            }
+
+        }
+    }
+
+    private void registerApartment() {
+        System.out.print("Enter price: ");
+        double price = scanner.nextDouble();
+
+        apartmentService.register(price);
+        System.out.println("\nApartment successfully registered");
+    }
+
+
+    private void reserveApartment() throws ReservationException {
+        System.out.print("Enter client name: ");
+        String clientName = scanner.nextLine();
+
+        Client client = new Client(clientName);
+        boolean reserved = apartmentService.reserve(client);
+        if (!reserved) {
+            throw new ReservationException("No available apartments found.");
+        }
+
+        System.out.println("\nApartment reserved successfully for " + client);
+    }
+
+    private void releaseApartment() throws ReservationException {
+        System.out.print("Enter apartment's id: ");
+        int id = scanner.nextInt();
+        boolean released = apartmentService.release(id);
+        if (!released) {
+            throw new ReservationException("No apartments found with such id.");
+        }
+        System.out.println("\nApartment successfully released");
```

Comment:
\n why we have it in each sout?

Reply:
when i was testing, thought it would be better to make a distance between the lines to make the result more readable